hack/test-tools.sh

```diff
@@ -6,7 +6,8 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 os::test::junit::declare_suite_start 'tools'
 
+os::cmd::expect_success 'hack/build-go.sh tools/junitreport'
```

Comment:
This line won't be necessary with the call to `os::util::ensure::built_binary_exists`